src/App.css

```diff
@@ -16,13 +16,13 @@
 .container {
   display: flex;
   flex-direction: column;
-  min-width: 25% ;
+  min-width: 30% ;
 }
 
 .logo {
   margin: 0.5em;
   color: #222;
-  font-weight: 300;
+  font-weight: 700;
 }
 
 /* Stat Cards */
```

Comment:
In `stat-heading` class name, remove `font-weight: 300` and let me know your thoughts.

Instead of using `em`, use `rem`